docs/rfcs/RFC0002-MonoRepo-Gardener-Rotations.md

```diff
@@ -0,0 +1,137 @@
+# Mono-repo Gardener
+
+Gardeners<sup>[1]</sup> are responsible for keeping their mono-repos shippable;
```

Comment:
Do we want to qualify gardening with "mono-repo", especially when there are multiple "mono" repos? Maybe we could pick a different label like "component group build gardeners".